Utilities/Files.cs

```diff
@@ -4,11 +4,17 @@
 using System.Security;
 using System.Text;
 using System.Threading;
+using System.Threading.Tasks;
 
 namespace Utilities
 {
     public class Files
     {
+        private const int readAttempts = 10;
+        private const int readIODelayMs = 25;
```

Comment:
Ms is technically "megaseconds" in the SI system, because the short names are case-sensitive. Recommend readIODelayMilleSecs and similarly for write.